src/System.Text.Json/System/Text/Json/JsonTokenType.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Text.Json
+{
+    public enum JsonTokenType
+    {
+        None,
+        StartObject,
+        EndObject,
+        StartArray,
+        EndArray,
+        PropertyName,
+        Comment,
+        Value,
+        Null,
+        Undefined,
```

Comment:
I'm leaving this in. It is implemented and adds value. Not worth deleting unless we decide we aren't doing it when we move to release.